weblate_web/templates/snippets/service.html

```diff
@@ -1,107 +1,167 @@
+                                    {% else %}
+                                    <t class="pl-1 grey">{{ subscription.expires|days_diff_from_today}} days remaining</t>
+                                    {% endif %}
+                                </span>
+                            </t>
+                            <br>
+                            <v style="font-size: 16px; color: #BFC3C7;">Note:
+                                <span style="color:#2C3548 ;">
+                                    {% if service.note %}
+                                    {{ service.note }}
+                                    {% endif %}
+
+                                </span>
+                            </v>
+
                             {% if service.status == "shared" %}
                             <a class="button small inline" href="https://hosted.weblate.org/create/billing/?billing={{ service.hosted_billing }}&amp;upgrade=1">{% trans "Change plan" %}</a>
                             {% endif %}
                         </div>
                         <div class="clear"></div>
                     </div>
                     {% if subscription.get_repeat %}
-                    <div class="form-line">
+                    <div class="form-line style6">
                         <div class="line-left">{% trans "Validity" %}</div>
-                        <div class="line-right">{% blocktrans with created=subscription.created|date:"SHORT_DATE_FORMAT" expires=subscription.expires|date:"SHORT_DATE_FORMAT" context "Date range" %}{{ created }}–{{ expires }}{% endblocktrans %}</div>
-                        <div class="clear"></div>
-                    </div>
-                    {% else %}
-                    <div class="form-line">
-                        <div class="line-left">{% trans "Creation date" %}</div>
-                        <div class="line-right">{{ subscription.created|date:"SHORT_DATE_FORMAT" }}</div>
+                        <div class="line-right">{% blocktrans with created=subscription.created|date:"d M Y" expires=subscription.expires|date:"d M Y" context "Date range" %}{{ created }}–{{ expires }}{% endblocktrans %}</div>
                         <div class="clear"></div>
                     </div>
                     {% endif %}
                     {% if subscription.get_repeat %}
-                    <div class="form-line">
-                        <div class="line-left">{% trans "State" %}</div>
+                    <div class="form-line style6">
+                        <div class="line-left">{% trans "Billing" %}</div>
                         <div class="line-right">
                             {% if not subscription.active %}
                             {% trans "Expired" %}
                             {% elif subscription.payment_obj.recurring %}
-                            {% trans "Automatic renewal" %}<br />
+                            {% trans "Automatic monthly renewal" %}<br />
```

Comment:
Please use the original string (it can be yearly renewal as well).

Reply:
It's done!